src/seedu/addressbook/commands/Command.java

```diff
@@ -45,6 +45,11 @@ public CommandResult execute(){
 
     //Note: it is better to make the execute() method abstract, by replacing the above method with the line below:
     //public abstract CommandResult execute();
+    
+    public boolean isMutating() {
```

Comment:
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.